### ISSUE TITLE
Spatialite Version is 4.3.0-RC1

### ISSUE THREAD

**Issue by mgiana** [gisinternals/buildsystem#102]
Hi,

I have downloaded the latest stable version for (15/11/2016)
release-1800-gdal-2-1-mapserver-7-0 and 
release-1800-x64-gdal-2-1-mapserver-7-0

why is it that the spatialite version is spatialite_version(): 4.3.0-RC1, where as with OSGeo4W its at 4.3.0. Specifically its missing the function ST_Azimuth as its part of LWGEOM, which seems to be missing from this build as I queried HasLwGeom() and it returns 0.

Are there plans to upgrade the spatialite component of the compiles gdal library?

Regards

Marco

**Comment by szekerest** [comment]
Marco,

I've checked the version I'm using and it is the same as what can be downloaded from:
http://www.gaia-gis.it/gaia-sins/libspatialite-sources/

(the latest sable version is libspatialite-4.3.0a.zip)

By looking into the package the config-msvc.h contains:  
#define PACKAGE_STRING "libspatialite 4.3.0-RC1"

I'm not sure where to obtain a more recent stable version.

Best regards,

Tamas

**Comment by mgiana** [comment]
Hi Tamas,

thanks for checking. I got around it by installing release-1500-gdal-2-1-mapserver-7-0 and release-1500-x64-gdal-2-1-mapserver-7-0 then replacing spatialite.dll with the one from OSGeo4W, which seems to be version 4.3.0-3

![image](https://cloud.githubusercontent.com/assets/1445815/20408085/1f0757b6-ad68-11e6-9b12-1a0fe1e621ac.png)

I am unsure where this comes from, but I assume its stable as I have not installed any of the dev gdal libraries. Anyways I have found a way around my issue, thank you for your help.
